packages/edge/docs/interfaces/ExtraResponseInit.md

```diff
@@ -66,4 +66,4 @@ Omit.statusText
 
 #### Defined in
 
-node_modules/typescript/lib/lib.dom.d.ts:1579
+node_modules/.pnpm/typescript@4.7.4/node_modules/typescript/lib/lib.dom.d.ts:1579
```

Comment:
Might want to get approval from the edge team on this change since this looks unexpected